projects/polynomial-trade/index.js

```diff
@@ -4,18 +4,31 @@ const { get } = require('../helper/http');
 // api
 const BASE_URL = "https://perps-v2-mainnet.polynomial.fi/snx-perps/tvl";
 
-async function tvl (timestamp, ethBlock) {
+const BASE_URL_POLYNOMIAL_CHAIN = "https://perps-api-mainnet.polynomial.finance/core/portfolio/tvl";
```

Comment:
what is included as tvl here

Reply:
we have smart contract wallet for every user who comes to polynomial trade on polynomial chain they can deposit money in this wallet, and they can deposit as margin in the protocol, we sum up these for all users and return in this api

Reply:
makes sense @g1nt0ki ?

Reply:
sorry, dont think you should count the tvl in smart wallet only the margin deposited in the protocol

Reply:
Hey @g1nt0ki i discussed with team, and we removed total balance in there smart contract and only agg the margin we deployed and the api should only show the total of margin of all accounts now

Reply:
thanks @Aniket965